webapp/helpers.py

```diff
@@ -27,3 +30,38 @@ def get_pagination_page_array(page, total_pages):
         )
 
     return list(range(first_page_to_show, last_page_to_show + 1))
+
+
+def get_download_url(model, model_details):
+    """
+    Return the appropriate ubuntu.com/download url for the model
+
+    :param model: a certifiedmodel resource
+    :param model_details: a certifiedmodeldetails resource
+    :return: appropriate ubuntu.com/download url
+    """
+    platform_category = model.get("category")
+    architecture = model_details.get("architecture")
+
+    if model_details.get("level") == "Enabled":
+        # Enabled systems use oem images without download links.
+        return
+
+    if re.search(r"desktop|laptop", platform_category, re.IGNORECASE):
+        return "https://ubuntu.com/download/desktop"
+
+    if re.search(r"core", platform_category, re.IGNORECASE):
+        return "https://ubuntu.com/download/iot/"
+
+    if re.search(r"server", platform_category, re.IGNORECASE):
+        # Server platforms have special landing pages based on architecture.
+        arch = ""
+        if re.search(r"arm", architecture, re.IGNORECASE):
```

Comment:
Wouldn't `"arm" == architecture.lower()` or `"arm" in architecture.lower()` be both syntactically and computationally simpler?